### ISSUE TITLE
Application.FunctionalTests small issue

### ISSUE THREAD

**Issue by przeslo** [jasontaylordev/CleanArchitecture#964]
**Describe the bug**
The issue pertains to the compilation of the `Application.FunctionalTests.csproj` file. By default, it does not compile successfully. The problem lies in the requirement to modify every instance of class `Testing` usage from:
`using static Testing;`
to
`using static CleanArchitecture.Application.FunctionalTests.Testing;`

Additionally, the `Application.FunctionalTests.csproj` file still contains the outdated namespace `IntegrationTests` within the `PropertyGroup` tag:

```
<PropertyGroup>
    <RootNamespace>CleanArchitecture.Application.IntegrationTests</RootNamespace>
    <AssemblyName>CleanArchitecture.Application.IntegrationTests</AssemblyName>
</PropertyGroup>
```

**To Reproduce**
1. Go to `Application.FunctionalTests`
2. Select any class that attempts to utilize `Testing.cs`

**Comment by jasontaylordev** [comment]
Thanks @przeslo, I am working on this now. I've fixed the outdated namespace, but I can reproduce the error with Testing.cs. I can build the solution and run the tests successfully. Are you able to share more about the issue? For example:

- What SDK version do you have installed?
- What IDE and version are you using?
- What version of the template are you using?
- What type of solution did you create? e.g. Angular, React, Web API only

Thanks in advance! 😀

**Comment by przeslo** [comment]
Alright, I delved a bit further and found a distinction. When I create a project using the console, indeed, the solution with the tests compiles correctly. However, if you create a project using the template available through the UI in Visual Studio, you'll notice an incorrectly added `using static Testing;` statement above the namespace instead of below it.

**Comment by jasontaylordev** [comment]
Nice find, @przeslo. I gave it a try and can see you're right. I guess this is a bug with Visual Studio. I noticed that it also drops the solution folders (src and test):

![image](https://github.com/jasontaylordev/CleanArchitecture/assets/1988321/0f658e3c-278a-423f-a31d-c4c0c2bd2589)

I'm going to raise an issue with the VS team to see if it can be resolved. I could, of course, fully namespace the testing class as suggested above, but I prefer the original approach and I would still not be happy without the solution folders.
